constraints.txt

```diff
@@ -0,0 +1 @@
+snowballstemmer<3.0.0  # incompatible with Twisted 7.4.7?
```

Comment:
Twisted 7.4.7? 7.4.7 is the sphinx version we're running

Reply:
I noticed that too..

Reply:
:sweat_smile: Force-pushed fix :)